### PR TITLE
Added encryptedSeed field to user

### PR DESCRIPTION
**Pull request by danbar0kal**

### CODE REVIEW

File: api_v3/lib/types/KalturaUser.php [hunk 1]
```diff
@@ -100,6 +100,11 @@ class KalturaUser extends KalturaBaseUser
 	 * @var string
 	 */
 	public $ksPrivileges;
+	
+	/**
```
Comment (MosheMaorKaltura) [inline api_v3/lib/types/KalturaUser.php:104]: Add readonly annotations

**Review by MosheMaorKaltura — COMMENTED** [review]
1. No need to save the encryptedSeed  on the core object since the seed exist.
2. Set is not needed on core object since its a read only value
3. The getter needs to be able to calculate the key on the spot and since it happens only once no scale issue is relevant.

File: alpha/lib/model/kuser.php [hunk 2]
```diff
@@ -27,6 +27,10 @@ public function __construct()
 	const COMPANY = 'company';
 
 	const CUSTOM_DATA_KS_PRIVILEGES = 'ks_privileges';
+	
+	const CUSTOM_DATA_ENCRYPTED_SEED = 'encryptedSeed';
+	const CIPHER_IV = '0000000000000000';
```
Comment (MosheMaorKaltura) [inline alpha/lib/model/kuser.php:32]: delete

**Comment by MosheMaorKaltura** [comment]
reviewed

**Review by gonenradai — APPROVED** [review]

**Pull request merged**